source/_integrations/reolink.markdown

```diff
@@ -18,6 +18,8 @@ ha_dhcp: true
 
 The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
 
+If you appreciate the reolink integration and want to support its development, please consider becoming a [![](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=%23fe8e86)](https://github.com/sponsors/starkillerOG) of the upstream library, or purchase Reolink products through [this affiliate link](https://reolink.pxf.io/q44QWq). 
```

Comment:
We will not allow external sponsor/affiliate links at this point in time.

The reason is that we want to prevent the chaos of everybody trying to get theirs in. Maybe we will reconsider this in the future.

Reply:
Thanks for letting me know the decision

Reply:
May I suggest adding this to the Home Assistant documentation [standards](https://developers.home-assistant.io/docs/documenting/standards), that was the first place I looked to figure out if this was allowed or not.

Reply:
If it comes up more frequently, I think that would make sense. This is the first occurrence, so I don't think it is needed at this point.

../Frenck